apps/frontend/src/components/TimeCardPage/CellTypes/CommentModals/ShowCommentModal.tsx

```diff
@@ -174,8 +201,18 @@ export default function ShowCommentModal({
       };
   
       const handleSubmit = () => {
-        // TODO: reuse comment validation
-        setComments([...comments, createNewComment(user, CommentType.Comment, remark)]);
+        
+        var rowToMutate = row.Comment; 
+        if (rowToMutate === undefined) {
+            rowToMutate = comments;
+        }
+        // Value is null, so mark it as undefined in our processing 
```

Comment:
Not sure what this comment is saying - I think this might be vestigial?

Reply:
8940a208c9a8217f7b754235085a00f9fc86d766